align_slice.py

```diff
@@ -28,3 +28,27 @@ def align_and_slice(s, dimension='t2', convwidth=500, threshold=0.05, fl=None):
     if fl is not None:
         fl.pop_marker()
     return s[dimension:slices]
+def correlation_align(s,avg,convwidth=0,dimension='t2',fl=None):
+    assert not s.get_ft_prop(dimension), 'I want time-domain data'
+    assert not avg.get_ft_prop(dimension), 'I want time-domain data'
```

Comment:
why are we doing this twice? one with avg and one with s?

Reply:
yes, these are two different objects

Reply:
avg is the average along the coherence channels? mean_all_but('t2') excludes t2 from the averaging?

Reply:
also, we want or do not want the dimensions of avg and s to be the same?

Reply:
I don't remember offhand, but I think their ndshapes are different b/c the second is an average.  Here "dimension" is the name of the dimension along which the correlation alignment is performed -- honestly, I should have called it "axis" instead, so I just pushed a commit where I renamed it.  See [this page](https://en.wikipedia.org/wiki/Cross-correlation) if you want a sense of what this is doing